src/pfe/file-watcher/server/src/controllers/projectEventsController.ts

```diff
@@ -252,15 +255,17 @@ export async function updateProjectForNewChange(projectID: string, timestamp: nu
                         changedFilesMap.delete(projectID);
                         logger.logProjectTrace("Removing project " + projectID + " from changedFilesMap.", projectID);
                     }
+                    // A build getting triggerred. Delete the project from the timerMap
```

Comment:
very minor nit: missing `is` in between `build` and `getting` also `triggerred` should be `triggered`.

Reply:
updated